core/jobs/batch_jobs/manual_voice_artist_name_job.py

```diff
@@ -163,11 +169,7 @@ def extract_added_voiceovers_between_successive_snapshots(
 
             return voiceovers_added_in_this_version
         except Exception as e:
-            raise Exception(
-                'Failed to get newly added voiceover between exploration '
-                'snapshot versions %s and %s' % (
-                    old_snapshot_model.id, new_snapshot_model.id)
-            ) from e
+            raise Exception(e) from e
```

Comment:
Bit confused -- what is the point of the except clause here? How is the behaviour different then just omitting the try/except?

Reply:
Removed from here since the line 383 will handle this.